### PR TITLE
completed html-syntax//MichaelCardona

### PR DESCRIPTION
**Pull request by MichaelCardona**
Screenshot 2021-03-06 143921

### CODE REVIEW

File: html-syntax/index.html [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Syntax</title>
+  </head>
+  <img src="https://assets.pokemon.com/assets/cms2/img/pokedex/full/025.png">
```
Comment (BrettAlbright) [inline html-syntax/index.html]: One tiny thing here Michael, make sure to add alt text to the img tag!

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
small update needed!

File: html-syntax/index.html (hunk 1 above)
Comment (MichaelCardona) [reply html-syntax/index.html]: I got it to commit :)

**Review by Zhiwutian — APPROVED** [review]
👍

**Pull request merged**